src/Splat/Logging/IFullLogger.cs

```diff
@@ -113,14 +113,6 @@ public interface IFullLogger : ILogger
         /// <param name="argument">The argument for formatting purposes.</param>
         void Debug<TArgument>(IFormatProvider formatProvider, [Localizable(false)] string message, TArgument argument);
 
```

Comment:
Lets make the full logger derive from the `IAllocationFreeLogging` -- we got the base class to make it easier for users to adapt to our magnitude of new operators now..